README.md

```diff
@@ -151,6 +151,8 @@ plans, check out our [kernel support policy](docs/kernel-policy.md).
 - The [SendCtrlAltDel](docs/api_requests/actions.md#sendctrlaltdel) API request
   is not supported for aarch64 enabled microVMs.
 - Configuring CPU templates is only supported for Intel enabled microVMs.
+- If a CPU template is not used in x86_64, MSR_IA32_TSX_CTRL MSR may not retain
```

Comment:
Would it be possible to be more specific when the register would not retain its value?

Reply:
Well, we don't save/restore it, but if the guest never changes it, the restored value will match the original value. It only affects those who override the default. Not sure how to put it better.

Reply:
Maybe it would be useful to phrase it something like: 

``` suggestion
- If a CPU template is not used in x86_64, overwrites of `MSR_IA32_TSX_CTRL` MSR 
  will not be preserved after restoring from a snapshot.
```
?

Reply:
Yes, looks good to me, thanks for the suggestion. Updated.